dev/com.ibm.ws.jaxrs.2.0.client/resources/com/ibm/ws/jaxrs20/client/internal/resources/JAXRSClientMessages.nlsprops

```diff
@@ -48,3 +48,11 @@ failed_to_extract_saml_token_from_subject.useraction=Ensure that the server.xml
 failed_run_as_subject=CWWKW0706E: An exception occurred while attempting to get the RunAsSubject. The exception was: [{0}].
 failed_run_as_subject.explanation=An exception occurred while getting the RunAsSubject.
 failed_run_as_subject.useraction=Make sure the server is configured and started properly.
+
+warn_missing_mpjwt_token=CWWKW0707W: The [{0}] attribute in the [{1}] configuration is set to [{2}] but the MicroProfile JWT token is not available. The request will not contain an Authorization header with the token.
+warn_missing_mpjwt_token.explanation=The runtime failed to find the MicroProfile JWT token in the runAs subject. This error usually means that there was a problem with the authentication. Make sure that there were no errors encountered while authenticating a user by using MicroProfile JSON Web Token (JWT).
+warn_missing_mpjwt_token.useraction=See the user action for any prior error that is logged. Check the server logs for any additional error messages.
+
+warn_mpjwt_prop_service_notavail=CWWKW0708W: The MicroProfile JWT Propagation service is not available, runtime cannot access the token to include it in the Authorization header.
+warn_mpjwt_prop_service_notavail.explanation=The MicroProfile JWT Propagation service requires java version to be 8 or later. 
```

Comment:
- Capitalize Java.
- Also, instead of "Java version to be..." we can say "Java SE 8," "Java Platform, Standard Edition 8," "Java SE Development Kit 8," or "Java SE Runtime Environment 8" depending on which one you intended.

Example: The MicroProfile JWT Propagation service requires Java SE 8 or later.